lib/relation.gd

```diff
@@ -156,6 +168,10 @@ DeclareGlobalFunction("IdentityBinaryRelation");
 ##  consisting of the tuples collection <A>elms</A>.
 ##  This construction is similar to <Ref Func="GeneralMappingByElements"/>
 ##  where the source and range are the same set.
+##  <Example><![CDATA[
+##  gap> r:=BinaryRelationByElements(Domain([1..3]),[Tuple([1,2]),Tuple([1,3])]);
+##  <general mapping: <object> -> <object> >
+##  ]]></Example>
 ##  </Description>
 ##  </ManSection>
```

Comment:
How about also adding an example for `EmptyBinaryRelation`, e.g.
```
gap> EmptyBinaryRelation(3) = BinaryRelationOnPoints([ [], [], [] ]);
true
```

Reply:
Hi @fingolfin , thank you for the suggestion. I have now added this and also added Is Symmetric/Reflexive/Transitive BinaryRelation based on your comment above. 
I've pushed these changes to this branch (as well as fixing a missing `;`).